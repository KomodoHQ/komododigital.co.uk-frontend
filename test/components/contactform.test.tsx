import React from 'react';
import ContactForm from '../../src/components/contactform';
import renderer from 'react-test-renderer';

test('ContactForm element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<ContactForm data={data} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
