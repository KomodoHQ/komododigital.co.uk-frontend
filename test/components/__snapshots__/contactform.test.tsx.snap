// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm element is rendered 1`] = `
<div>
  <form
    action="https://formspree.io/hello@komododigital.co.uk"
    method="POST"
  >
    <label
      htmlFor="form-name"
    >
      Your Name
    </label>
    <input
      id="form-name"
      name="name"
      placeholder="Joe Blogs"
      type="text"
    />
    <label
      htmlFor="form-email"
    >
      Your Email
    </label>
    <input
      id="form-email"
      name="email"
      placeholder="joe@blogs.com"
      type="email"
    />
    <label
      htmlFor="form-service"
    >
      Service
    </label>
    <select
      id="form-service"
      name="service"
    >
      <option
        value="Discovery"
      >
        Discovery
      </option>
      <option
        value="Design"
      >
        Design
      </option>
      <option
        value="Development"
      >
        Development
      </option>
      <option
        value="Delivery"
      >
        Delivery
      </option>
    </select>
    <label
      htmlFor="form-sector"
    >
      Sector
    </label>
    <select
      id="form-sector"
      name="sector"
    >
      <option
        value="Education"
      >
        Education
      </option>
      <option
        value="Law Enforcement"
      >
        Law Enforcement
      </option>
      <option
        value="Startups"
      >
        Startups
      </option>
      <option
        value="Social Housing"
      >
        Social Housing
      </option>
    </select>
    <label
      htmlFor="form-message"
    >
      Your Message
    </label>
    <textarea
      cols={30}
      id="form-message"
      name="message"
      rows={10}
    />
    <input
      type="submit"
      value="Send"
    />
  </form>
</div>
`;
